languages/ruby/exercises/concept/numbers/.meta/design.md

```diff
@@ -0,0 +1,44 @@
+# Design
+
+## Goal
+
+The goal of this exercise is to teach the student how the concept of numbers is implemented in Ruby. It will introduce this concept through the two most common numeric types in Ruby: [`Integer`][integer-ruby] (whole number) and [`Float`][float-ruby] (floating-point number).
+
+## Learning objectives
+
+- Know of the existence of the two most commonly used number types, `Integer` and `Float`.
+- Understand that an `Integer` represents whole numbers, and a `Float` represents floating-point numbers.
+- Know of basic operators such as multiplication, comparison and equality.
+- Know how to conditionally execute code using an `if`, `unless` or `case` statement.
+
+## Out of scope
+
+- Any other numeric types besides `Integer` and `Float` (so no `Rational`, `BigDecimal`, etc.).
+- Parsing an instance of `String` to an instance of `Integer` or `Float`.
+- Converting an instance of `Integer` or `Float` to an instance of `String`.
+
+## Concepts
+
+The Concepts this exercise unlocks are:
+
+- `numbers`: know of the existence of the two most commonly used number types, `Integer` and `Float`; understand that the former represents whole numbers, and the latter floating-point numbers; know of basic operators such as multiplication, comparison and equality; know how to convert from one numeric type to another;
```

Comment:
```suggestion
- `numbers`: know of the existence of the two commonly used number types, `Integer` and `Float`; understand that the former represents whole numbers, and the latter floating-point numbers; know of basic operators such as multiplication, comparison and equality; know how to convert from one numeric type to another;
```

Reply:
I removed the last semicolon, as it seems to be out of place.

Reply:
Same comment as above, "coerce" is a tricky word for non-native speakers.